cmd/registry/cmd/upload/csv.go

```diff
@@ -57,7 +58,14 @@ func csvCommand(ctx context.Context) *cobra.Command {
 				log.FromContext(ctx).WithError(err).Fatal("Failed to get client")
 			}
 
-			core.EnsureProjectExists(ctx, adminClient, projectID)
+			if _, err := adminClient.UpdateProject(ctx, &rpc.UpdateProjectRequest{
```

Comment:
Do you think this is clearer? On the left I know exactly what is happening from the function name and on the right I have to parse 8 lines of text.

Reply:
I think they're both pretty clear, but I agree this is less concise so it loses some points there. On the other hand, there is value in being able to see what is happening in detail without having to look at a separate function. I consider this change a toss-up for readability overall.

We can discuss whether it would be better in a _local_ helper function (again though, I think it's a toss-up), but there are more important questions to address first.

- Should this code be here at all? None of our other commands create a project because we're encouraging flat organization of APIs into a single project per user.
- Given how many occurrences there are of similar code throughout the registry tool, shouldn't we have a more robust library instead of several mostly overlapping helper functions scattered throughout the codebase? I mention this in #445.

Reply:
OK, I see what you were doing as chipping away at core (which makes sense). I can see value in making this a local helper function for clarity but let's not make that change now. I'm also happy to see a "robust library" emerge, but would wait for a clear need for that. The "mostly overlapping helper functions" don't strike me as a large problem yet (when they become so, we can create the library)